Deploy/docker/Dockerfile.liquid

```
@@ -0,0 +1,22 @@
+FROM rust:1-bookworm AS builder
+WORKDIR /workspace
+
+RUN apt-get update && apt-get install -y --no-install-recommends protobuf-compiler libprotobuf-dev && rm -rf /var/lib/apt/lists/*
+ENV PROTOC_INCLUDE=/usr/include
+
+COPY Cargo.toml Cargo.toml
+{% if has_cargo_lock %}COPY Cargo.lock Cargo.lock
```

Hadolint (2.14.0)

[error] unexpected '{'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)